### PR TITLE
Override domainBaseUrl per Notion recommendation

### PR DESCRIPTION
**Pull request by laughnan**
Recommended fix from Ryo ([Notion on Reddit](https://www.reddit.com/r/Notion/comments/p3wgyn/is_it_just_happening_to_me_fruitionsitecom/h8vhdmn)) and @wentin in the [issue queue](https://github.com/stephenou/fruitionsite/issues/107).

Screen Shot 2021-08-13 at 23 02 16

### CODE REVIEW

File: src/code.js [hunk 1]
```diff
@@ -206,14 +206,15 @@ ${slugs
...
-  
+
   class BodyRewriter {
     constructor(SLUG_TO_PAGE) {
       this.SLUG_TO_PAGE = SLUG_TO_PAGE;
     }
     element(element) {
       element.append(\`<div style="display:none">Powered by <a href="http://fruitionsite.com">Fruition</a></div>
       <script>
+      window.CONFIG.domainBaseUrl = "https://\${MY_DOMAIN}";
```
Comment (laughnan) [inline src/code.js:217]: Apologies for the noise (my IDE cleaned up the whitespace). Here is the change for `code.js`

**Review by sweethuman — APPROVED** [review]

**Comment by qbrian** [comment]
the corrections work for me

**Comment by laughnan** [comment]
Closing this PR since @stephenou included it in their latest commit 🎉  https://github.com/stephenou/fruitionsite/commit/8b1f7dcf4e593348f3ba73a345f8b99b01dfc47d

**Comment by joshsabol46** [comment]
I'm still experiencing this as of Oct 4, 21.

Reported my issue here: https://github.com/stephenou/fruitionsite/issues/113#issuecomment-933634168

CC: @laughnan @sweethuman

**Pull request closed without merging**